README.md

```diff
@@ -49,7 +49,7 @@ You can see a (customized) instance of Strapping in action by visiting http://oV
 2. Clone the repository:
 
    ```
-   git clone https://github.com/OSAS/strapping-mediawiki strapping
+   git clone https://github.com/axxie/strapping-mediawiki strapping
```

Comment:
@axxie for future reference, that's a feature of github: if you edit a branch from which you created a PR, the PR is automatically updated. (You should see the message "Add more commits by pushing to the master branch on axxie/strapping-mediawiki" at the bottom of this page.)